content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -145,12 +112,14 @@ export class _CollapsibleSection extends React.PureComponent {
     this.setState({showContextMenu});
   }
 
+  onCtaClick(e) {
+    e.stopPropagation();
```

Comment:
why the stop propagation?

Reply:
It's to ensure when the link is clicked it doesn't also collapse the section.

I'm happy to fix this in a way where it doesn't use stop propagation though.